test/mario.spec.js

```diff
@@ -165,6 +164,9 @@ describe('mario', function () {
                                     touch: true,
                                     version: '10.0'
                                 });
+                                expect(mario(userAgentString), 'to not have key', {
+                                    windowsPhone: true
+                                });
```

Comment:
I think it should work if you just add windowsPhone: false to the existing assertion :-)

Reply:
I tried, but it doesn't work. 

Seems like it's failing the `to only have keys` assertion since there is no `windowsPhone` key for desktop touch string provided by mario. I suppose this test would pass if I explicitly export `windowsPhone:false` prop, but then it's not really following the logic of the library.

Any suggestions?

Reply:
Arh, so it's not set to false when not found, but just set to true when identified.

In that case, I think you should be able to do this:

```
windowsPhone: expect.it('to be undefined')
```

Reply:
I tested that, but it wouldn't work. First I need to update unexpected and then event with this evaluation it still checks for `windowsPhone` key presence. I did something else instead. Not so elegant, though, but allows to pass undefined value keys. Do you think it is something we can live with?